stripe/resource.py

```diff
@@ -55,6 +55,16 @@ def convert_to_stripe_object(resp, api_key, account):
         return resp
 
 
+def convert_array_to_dict(arr):
+    if isinstance(arr, list):
+        d = {}
+        for i, value in enumerate(arr):
```

Comment:
Yeah, I wanted to do that as well (see the first commit). That syntax doesn't work in Python 2.6